ejb3/src/main/java/org/jboss/as/ejb3/component/stateful/StatefulComponentDescription.java

```diff
@@ -431,8 +431,8 @@ public void setPassivationApplicable(final boolean passivationApplicable) {
 
     /**
      * EJB 3.2 spec allows the TimeService to be injected/looked up/accessed from the stateful bean so as to allow access to the {@link javax.ejb.TimerService#getAllTimers()}
```

Comment:
EJB 3.2 spec allows the TimerService to be ...

Reply:
I didn't touch this, but will fix.